src/common/function/FunctionManager.cpp

```diff
@@ -2799,6 +2801,21 @@ FunctionManager::FunctionManager() {
       }
     };
   }
+  // Get any argument which is not empty/null
+  {
+    auto &attr = functions_["_any"];
+    attr.minArity_ = 1;
+    attr.maxArity_ = INT64_MAX;
+    attr.isAlwaysPure_ = true;
+    attr.body_ = [](const auto &args) -> Value {
+      for (const auto &arg : args) {
+        if (!arg.get().isNull() && !arg.get().empty()) {
+          return arg.get();
+        }
+      }
+      return Value::kNullBadData;
```

Comment:
`kNullBadData` is designed for error and`kNullValue` is sufficient for ignoring behavior. Fwiw, the `bad null` should be passed through by all functions or expressions.

Reply:
What' your meaning?